contributing.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to {{Project-Name}}
+
+## Where to get help
+The main way to get help is on our [discord server](https://discord.gg/uh69TdKfBD).  Though, this guide should help you get started. It may be slightly lengthy, but it designed for those who are new so please don't let length intimidate you.
+
+## Code of Conduct
+Please be nice to each other when interacting with others and follow our [code of conduct](<link to code of conduct>).
+
+## How to contribute
+In order to contribute you can either add a new issue or you can create a pull request and link it to an existing issue. If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it.
+
+### Contributing by adding an issue
+If you have found a bug and would like to report it or if you have a feature that you feel we should add, then it would be appreciated if you contributed by adding an issue. Before you do so please search the other issues to ensure that you are not attempting to add an issue that has already been created. In order to submit a new issue just hit the issue button and a choice between two templates should appear and just follow along with the template you chose. If you don't know how to fill in all parts of the template just go ahead and skip those parts. You can edit the issue later.
+
+### Contributing through code
+In order to contribute through code follow the steps below. Note that you don't need to be the best programmer to contribute. Our discord is open for questions
+
+ 1. Pick a feature you would like to add or a bug you would like to fix. 
+	 - If you wish to contribute but what you want to fix/add is not already listed in the issues go ahead and add the issue
```

Comment:
> If you wish to contribute but what you want to fix/add is not already ~~listed in the issues go ahead and add the issue~~ covered in an existing issue, please open a new issue 

Suggested rewording.

Also, we may want another bullet point here encouraging folks to discuss the bug or feature before they start coding.